core/embed/trezorhal/display.h

```diff
@@ -17,11 +17,12 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#ifndef _DISPLAY_INTERFACE_H
-#define _DISPLAY_INTERFACE_H
+#ifndef TREZORHAL_DISPLAY_H
+#define TREZORHAL_DISPLAY_H
 
 #include <stdint.h>
 #include "common.h"
+#include "display_draw.h"
```

Comment:
i know we talked about this before, but looking at it closer now i dislike this already, lets remove this from here and add it where necessary, its just few places and shouldn't take much effort.